### ISSUE TITLE
Models are overfit to a fixed noise pattern

### ISSUE THREAD

**Issue by rcrandall** [cswin/WIN#1]
The noise pattern used in testing is fixed, because randn('seed',0); is always called before adding the noise: https://github.com/cswin/WIN/blob/master/Demo_test_WIN5RB.m#L51

Further, the input image is always resized to 321x481 inside Consistency.m. This means that the exact same noise pattern is used in every test. The accuracy drops significantly if I comment out randn('seed',0), or if I resize the input image to something other than 321x481. This suggests the provided CNN models are overfit to encode this specific noise pattern.

**Comment by 13331151** [comment]
And they do the rand seed also in generating the training data, I think the performance provided is questionable.
